Makefile

```
@@ -4,6 +4,7 @@ build:
 	env GOOS=linux GOARCH=amd64 go build -ldflags="-s -w" -o bin/hello hello/main.go
 	env GOOS=linux GOARCH=amd64 go build -ldflags="-s -w" -o bin/dynamodb-example dynamodb-example/main.go
 	env GOOS=linux GOARCH=amd64 go build -ldflags="-s -w" -o bin/messages messages/main.go
+	env GOOS=linux GOARCH=amd64 go build -ldflags="-s -w" -o bin/notifications notifications/main.go
```

after you rebase your code on master, you'll see it should become this
```suggestion
	env GOOS=linux GOARCH=amd64 go build -ldflags="-s -w" -o bin/notifications lambda/notifications/main.go
```